src/app/product/product.component.html

```diff
@@ -1 +1,38 @@
-<p>product works!</p>
+<h3>Products</h3>
+
+
+<form [formGroup]="form" (ngSubmit)="sell()">
+  <mat-form-field>
+    <mat-label id="name">Name</mat-label>
+    <input formControlName="name" matInput type="text" required>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-label id="description">Description</mat-label>
+    <textarea formControlName="description" matInput maxlength="300" required></textarea>
+    <mat-hint>300 Characters max</mat-hint>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-label id="price">Price</mat-label>
+    <input matInput formControlName="price" type="number" min="0" step="0.01" pattern="^\d*(\.\d{0,2})?$" required>
```

Comment:
Depending on the product we might want to put a higher step, except if this is a clicker game ;)